hal/halx86/acpi/halacpi.c

```diff
@@ -230,7 +230,7 @@ HalpAcpiGetTableFromBios(IN PLOADER_PARAMETER_BLOCK LoaderBlock,
             if (!Xsdt) return NULL;
 
             /* Won't be using the RSDT */
-            Rsdt = NULL;
+            // Rsdt = NULL; // Redundant.
```

Comment:
Code does not need to document its history, that's what Git is for. Just delete the assignment (or leave it there, since it doesn't cause any problem).

Reply:
Deleted.